.github/workflows/release.yaml

```diff
@@ -38,3 +47,25 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           COSIGN_PASSWORD: ${{ secrets.COSIGN_PASSWORD }}
           COSIGN_PRIVATE_KEY: ${{ secrets.COSIGN_PRIVATE_KEY }}
+
+      - name: Finish and Record Attestation
+        if: ${{ success() }}
+        run: |
+          chainloop attestation status --full
+          chainloop attestation push --key env://CHAINLOOP_SIGNING_KEY
+        env:
+          CHAINLOOP_SIGNING_KEY: ${{ secrets.COSIGN_PRIVATE_KEY }}
+          CHAINLOOP_SIGNING_PASSWORD: ${{ secrets.COSIGN_PASSWORD }}
```

Comment:
I'm sure you know what you're doing, but how is this being used? Is it a variable `chainloop attestation push` picks up on its own?

Reply:
Every `chainloop att` command uses the robot-account token to figure out what controlplane workflow this attestation refers to. 

This token can be provided either via the `--token` flag or the `CHAINLOOP_ROBOT_ACCOUNT` env variable. We do the latter here.

Would you prefer to make it explicit?

```
chainloop att        
Attestations related operations

Usage:
  chainloop attestation [command]

Available Commands:
  add         add a material to a workflow run attestation
  init        initialize workflow run
  push        generate and push the attestation to the control plane
  reset       cancel the current workflow run attestation
  status      check the status of the current workflow run attestation

Flags:
      --graceful-exit   exit 0 in case of error. NOTE: this behavior will change once 1.0 is reached (default true)
  -h, --help            help for attestation
  -t, --token string    robot account token. NOTE: You can also use the env variable CHAINLOOP_ROBOT_ACCOUNT
```